### PR TITLE
Added svg clock sample

### PR DESCRIPTION
**Pull request by Zaid-Ajaj**
This sample demonstrates the usage of svg in Elmish. However, I am not sure if I did the `Cmd` part idiomatically nor if there is a simpler way. Anyways here is it

elmish-svg-clock

### CODE REVIEW

File: samples/react/SvgClock/App.fs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+open Elmish
+open Elmish.React
+
+
+let tickTime() = async {
+    do! Async.Sleep 1000
+    return DateTime.Now
+}
+
+let tickCommand = Cmd.ofAsync tickTime () Tick (fun _ -> Tick (DateTime.Now))
```
Comment (et1975) [inline samples/react/SvgClock/App.fs]: This should be a subscription, there is a sample in the native that does it.

**Comment by et1975** [comment]
Neat, thank you!

**Comment by Zaid-Ajaj** [comment]
@et1975 Thanks! 
Could you please explain to me why this didn't work:
```fsharp
let initialState() = CurrentTime DateTime.Now, Cmd.none
let update (Tick next) (CurrentTime time) = CurrentTime next, tickCommand
``` 
but when I changed `Cmd.none`to `tickCommand` in the `initialState()` it did work:
```fsharp
let initialState() = CurrentTime DateTime.Now, tickCommand
let update (Tick next) (CurrentTime time) = CurrentTime next, tickCommand
```

If it is only in the initial state, why didn't the command in the update take care of subsequent "ticks" unless I have passed first in the initial state?

**Comment by et1975** [comment]
If there's no initial command, there's no call to `update`.

**Comment by Zaid-Ajaj** [comment]
@et1975 that makes sense, thanks a lot :smile:

**Comment by Zaid-Ajaj** [comment]
@et1975 Would you mind to take a look to see if it now looks good?

**Comment by et1975** [comment]
Sweet, thanks!

**Pull request merged**